lms/djangoapps/course_goals/views.py

```diff
@@ -89,33 +84,36 @@ def create(self, post_data):  # lint-amnesty, pylint: disable=arguments-differ
                 status=status.HTTP_400_BAD_REQUEST,
             )
 
-        user = post_data.user
+        manually_updated = request.data.get('manually_updated')
+        # So request.data returns a QueryDict object which converts all of the values to strings.
+        # I wasn't able to find anything about how to convert to boolean or just let it treat it as
+        # boolean... so here we are. I apologize in advance and if anyone can fix, please do.
+        if manually_updated in {'True', 'true'}:
```

Comment:
i think you could do something like
`manually_updated = bool(request.GET.get('manually_updated'))`

Reply:
Unfortunately not. `bool('False')` returns True because all strings are truthy.

Reply:
ah yes good point